packages/smart-wallet/CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## 0.3.0 (2022-09-20)
+
+
+### Features
+
+* **wallet:** more diagnostics for invitation match ([98630ee](https://github.com/Agoric/agoric/commit/98630ee96a202cf3907e37b5d4d549bb37b1263d))
+* **wallet-ui:** start displaying balances ([0f36da9](https://github.com/Agoric/agoric/commit/0f36da99daef86f24670d606ae5fd1adb32b419b))
+* ensure voting via PSMCharter works with a unit test ([#6167](https://github.com/Agoric/agoric/issues/6167)) ([ff9471b](https://github.com/Agoric/agoric/commit/ff9471bf3a90ffab050e8b659d64d4cbd7c2d764))
```

Comment:
@Chris-Hibbert was this meant to be published as Feature news?

Reply:
I didn't have very coherent intentions. I try to follow the rules about conventional commits, but I don't have a clear model about how that feeds into these reports, or what I should be trying to ensure they contain.

What's surprising to you about this entry? It looks like the other entries nearby AFAICT.  It's moderately surprising that it showed up in **smart-wallet**, but maybe that's because it didn't specify a component, so it showed up in several locations.

Reply:
> What's surprising to you about this entry?

That it's listed in **Features** . I read the PR title as about testing. So per ConCom it would be "test: ensure voting via PSMCharter works". But reading the description I see "this unit test demonstrates (and makes it be the case) that votes can be instigated and carried out through the PSM Charter" so I would expect a master commit "feat: PSM Charter voting".

> It's moderately surprising that it showed up in smart-wallet

The changelog maker looks at the git history for each package and the PR's squash commit touched several. Here's a file touched in smart-wallet:
 https://github.com/Agoric/agoric-sdk/pull/6167/files#diff-332a7606503ad801c4772a3670bbe4b86a865f156b1a04d3c5ef34dcb5a342c5

Reply:
```
* ensure voting via PSMCharter works with a unit test ([#6167](https://github.com/Agoric/agoric/issues/6167)) ([ff9471b](https://github.com/Agoric/agoric/commit/ff9471bf3a90ffab050e8b659d64d4cbd7c2d764))
```
There seem to be several (6?) commits squashed together here. The top one is titled "feat: ensure voting via PSMCharter works with a unit test", which seems a fair characterization to me. The unit test showed that one path through the code would work correctly, without ensuring that other paths were complete. (We later did further tests using the CLI, and found more things to fix.)

Would you recommend not mentioning the test in the title?

Reply:
> Would you recommend not mentioning the test in the title?

If that's not the newsworthy change, no. But it's the "ensure" that I think makes it sound like it was already working, and now there's coverage.

Higher level, I wonder about whether conventional commits are worth the effort. There are other approaches to changelogs, like https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

Reply:
> Higher level, I wonder about whether conventional commits are worth the effort.

I'm not getting any value out of it, but probably someone else is. The current approach is relatively low-cost to me, as long as the way I'm doing it isn't mucking things up.

> There are other approaches to changelogs, like https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

I'm not opposed to trying something different.